src/genericComponents/components.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useRef } from "react";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+import Home from "../pageComponents/Home";
+import Film from "../pageComponents/Film";
+import Serietv from "../pageComponents/Tvseries";
+import logo from "../images/lg.png";
+
+//page structure component
+export default function PageStructure() {
+  return (
+    <div>
+      <UpperHeader />
+      <Menu />
+    </div>
+  );
+}
```

Comment:
Questo io lo inserirei direttamente in App.js togliendo di mezzo il componente pageStructure che fa da intermediario. (non è necessario)